docs/plugins/file_input.md

```diff
@@ -4,18 +4,19 @@ The `file_input` plugin reads logs from files. It will place the lines read into
-| `output`        | required | The connected plugin(s) that will receive all outbound entries                                                      |
-| `include`       | required | A list of file glob patterns that match the file paths to be read                                                   |
-| `exclude`       | []       | A list of file glob patterns to exclude from reading                                                                |
-| `poll_interval` | 200ms    | The duration between filesystem polls                                                                               |
-| `multiline`     |          | A `multiline` configuration block. See below for details                                                            |
-| `write_to`      | $        | A [field](/docs/types/field.md) that will be set to the log message                                                 |
-| `path_field`    |          | A [field](/docs/types/field.md) that will be set to the path of the file the entry was read from                    |
-| `start_at`      | `end`    | At startup, where to start reading logs from the file. Options are `beginning` or `end`                             |
-| `max_log_size`  | 1048576  | The maximum size of a log entry to read before failing. Protects against reading large amounts of data into memory. |
+| Field             | Default  | Description                                                                                                         |
+| ---               | ---      | ---                                                                                                                 |
+| `id`              | required | A unique identifier for the plugin                                                                                  |
+| `output`          | required | The connected plugin(s) that will receive all outbound entries                                                      |
+| `include`         | required | A list of file glob patterns that match the file paths to be read                                                   |
+| `exclude`         | []       | A list of file glob patterns to exclude from reading                                                                |
+| `poll_interval`   | 200ms    | The duration between filesystem polls                                                                               |
+| `multiline`       |          | A `multiline` configuration block. See below for details                                                            |
+| `write_to`        | $        | A [field](/docs/types/field.md) that will be set to the log message                                                 |
+| `path_field`      |          | A [field](/docs/types/field.md) that will be set to the path of the file the entry was read from                    |
```

Comment:
If we're going with `file_name_field`, maybe `path_field` should be `file_path_field` instead.